examples/src/main/scala/mhtml/examples/HelloWorld.scala

```diff
@@ -1,4 +1,5 @@
 package mhtml.examples
+import mhtml._
```

Comment:
I don't know, I think it's weird to have to import `mhtml._` to use `scala.xml._`, putting them in the companion object removes the need for wild card import entirely, so I would say "fuck it", we are too deep into a hack to care about modularity :P